### PR TITLE
[UXENG-4] Flexbox helper class documentation

### PR DESCRIPTION
**Pull request by Elepert**
## Summary

Added section in Storybook on `Flexbox` helper classes.

Screen Shot 2020-07-01 at 2 06 44 PM

## JIRA

[UXENG-4](https://optimizely.atlassian.net/browse/UXENG-4)

### CODE REVIEW

File: src/oui/_oui-partials.scss [hunk 1]
```diff
@@ -6,46 +6,46 @@
 ////
 
 /// Placeholders and mixins
-@import 'partials/sass/placeholders';
-@import 'partials/sass/mixins';
+@import "partials/sass/placeholders";
```
Comment (ayeshakmaz) [inline src/oui/_oui-partials.scss:9]: Are the changes in this file necessary?

File: .storybook/helper_classes/index.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+// 2. flex-wrap
+// 3. flex-self--
+// ?  flex-overflow-fix still needed?
+/// Axiom Tokens
+@import "../../src/tokens/forimport/index";
+
+body,
+html {
+  overflow: auto;
+}
```
Comment (ayeshakmaz) [inline .storybook/helper_classes/index.scss:11]: Is this necessary?

File: .storybook/helper_classes/index.scss [hunk 3]
```diff
@@ -0,0 +1,76 @@
+// 1. flex--none in the app?
+// 2. flex-wrap
+// 3. flex-self--
+// ?  flex-overflow-fix still needed?
```
Comment (ayeshakmaz) [inline .storybook/helper_classes/index.scss:4]: Can you remove these

File: .storybook/helper_classes/flexboxInteractive.tsx [hunk 4]
```diff
@@ -0,0 +1,519 @@
...
+                <div className="push--bottom">
+                    <TabNav activeTab={flexTab} style={['sub']}>
+                        <TabNav.Tab onClick={() => this.switchFlexTab('align')} tabId="align">
+                            flex-align--(start, center, end)
+                        </TabNav.Tab>
+                        <TabNav.Tab onClick={() => this.switchFlexTab('self')} tabId="self">
+                            flex--self-(start, center, end)
+                        </TabNav.Tab>  
+                        <TabNav.Tab onClick={() => this.switchFlexTab('justified')} tabId="justified">
+                            flex-justified--(start, center, end)
```
[the remark this answers is not in the document]
Comment (Elepert) [reply .storybook/helper_classes/flexboxInteractive.tsx:405]: I made it a separate tab bc it was getting a bit crowded.

File: src/oui/_oui-partials.scss (hunk 1 above)
Comment (Elepert) [reply src/oui/_oui-partials.scss:9]: Ah, this is leftover from when my linter was being annoying.

File: .storybook/helper_classes/index.scss (hunk 2 above)
Comment (Elepert) [reply .storybook/helper_classes/index.scss:11]: Nope, leftover from the CodePen.

File: .storybook/helper_classes/index.scss (hunk 3 above)
Comment (Elepert) [reply .storybook/helper_classes/index.scss:4]: Nope, leftover from the CodePen.

**Pull request merged**